docs/data/sql_functions.yml

```diff
@@ -83,7 +83,7 @@ logical:
     table: BOOLEAN1 && BOOLEAN2
     description: Returns TRUE if BOOLEAN1 and BOOLEAN2 are both TRUE. Supports three-valued logic. E.g., true && Null(BOOLEAN) returns UNKNOWN.
   - sql: NOT boolean
-    table: '!BOOLEAN'
+    table: BOOLEAN.not() or '!BOOLEAN' (Scala only)
```

Comment:
`and` and `or` are exposed both as infix and postfix. I went with postfix only for now here.